qutip/core/superoperator.py

```diff
@@ -60,7 +60,7 @@ def out(qobj):
     return out
 
 
-def liouvillian(H, c_ops=[], data_only=False, chi=None):
+def liouvillian(H=None, c_ops=None, data_only=False, chi=None):
```

Comment:
We should update the docstring to explain what happens when `H` and `c_ops` are `None`.

Reply:
And probably what `chi` and `data_only` are while we're here.

Reply:
I can't really explain the `chi`...
There are a few thing I don't understand with `lindblad_dissipator` (not trace preserving when `b` is given).
Now that `Qobj` creation is fast, `data_only` could be removed, but adding a description for now.